jest.config.js

```diff
@@ -1,5 +1,5 @@
 const config = {
-  testEnvironment: 'node',
+  testEnvironment: 'jsdom',
```

Comment:
I believe this is the default one, we shouldn't have to change it

Reply:
I had to change this because React Router and JS host objects were complaining.

Reply:
We should not be testing React Router code though (which leave in containers). The code we test is universal